_posts/en/newsletters/2019-11-20-newsletter.md

```diff
@@ -0,0 +1,163 @@
+      to add support for the corresponding [BIP157][] protocol that will
+      allow sharing these filters with clients over the P2P network.
+
+    - **Deprecated or removed features:** support for the [BIP70][]
+      payment protocol, [BIP37][] P2P protocol bloom filters, and
+      [BIP61][] P2P protocol reject messages has been disabled by
+      default, eliminating the source of various problems (respectively,
+      see Newsletters [#19][news19 bip70], [#57][news57 bip37], and
+      [#37][news37 bip61]).  The payment protocol and reject messages
+      are scheduled to be fully removed in the next major Bitcoin Core
+      version about six months from now.
+
+    - **Customizable permissions for whitelisted peers:** when
+      specifying which peers or interfaces should be whitelisted, users
+      can now specify which special features the whitelisted peers
+      can access.  Previously, whitelisted peers wouldn't be banned and
+      received relayed transactions faster.  These defaults haven't
+      changed, but now it's possible to toggle those settings on a
+      per-peer basis or to allow specified whitelisted peers to request
+      BIP37 bloom filters even though they're disabled for
+      non-whitelisted peers by default.  For details, see [Newsletter
+      #60][news60 16248].
+
+    - **GUI improvements:** graphical users can now create new wallets
+      for use with multiwallet mode from the GUI's *file* menu (see
+      [Newsletter #63][news63 new wallet]).  The GUI also now provides
+      users with [bech32][topic bech32] Bitcoin addresses by default,
+      but users can easily request a backwards-compatible P2SH-P2WPKH
+      address by toggling a checkbox next to the button to generate an
+      address (see [Newsletter #42][news42 core gui bech32]).
+
+    - **Optional privacy-preserving address management:** a new
+      `avoid_reuse` wallet flag, which can be toggled using a new
+      `setwalletflag` RPC, can be enabled to prevent the wallet from
+      spending bitcoins received to an address that was previously
+      used (see [Newsletter #52][news52 avoid_reuse]).  This prevents
+      certain privacy leaks based on block chain analysis such as [dust
+      flooding][].
+
+    For a full list of notable changes, links to the PRs where those
+    changes were made, and additional information useful to node
+    operators, please see the Bitcoin Core project's [release notes][notes
+    0.19.0].
+
+- **Mailing list downtime:** the Bitcoin-Dev and Lightning-Dev mailing
+  lists both experienced [downtime][bishop migration] last week due to an unannounced
+  server migration.  Both lists are functional again as of this writing.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
```

Comment:
I think the bug fix https://github.com/bitcoin/bitcoin/pull/17449 is notable too :)

I think it would be nice if the reporting contributor (`NullFunctor`) was credited. He/she allowed us to ship 0.19.0.* without an "optimisation unstable" `UpdateTip(…)` (luckily low impact AFAICT). Such behaviour should be celebrated! :)

Thanks NullFunctor! :)

Reply:
We usually don't report on bugfixes unless they result in a behavior change that might break downstream user code or workflows. That said, maybe I can briefly mention this in a note about 0.19.0.1 being the release version number, as requested by you and Jon.